index.rst

```diff
@@ -19,8 +19,8 @@ topics.
    :caption: Tutorials
 
    tutorials/tut-cluster.rst
-   tutorials/tut-real-time.rst
-   tutorials/tut-user-data.rst
+   tutorials/tut-hash-distribution.rst
+   tutorials/tut-append-distribution.rst
```

Comment:
This diff doesn't make it super clear, but the sections were renamed and then put in the opposite order.